app/src/test/java/tttTDD/Java/BoardTest.java

```diff
@@ -8,23 +8,25 @@ public class BoardTest {
     Board board;
 
     @Test
-    public void getBoardInstanceWithCorrectLength() {
-        board = new Board();
+    public void getBoardInstanceWithCorrectDefaultLengthOf3() {
+        board = new Board(3);
 
-        Assert.assertEquals(board.getBoard().length, 9);
+        Assert.assertEquals(board.getSpace(0), "1");
+        Assert.assertEquals(board.getSpace(8), "9");
```

Comment:
How do these two assertions prove that the board is the correct size?

Reply:
It asserts that the lower and upper bounds of the board exist. I can always add another to specify it did not go over the  total number of cells for a 3x3 board.

Reply:
I think to fully imply the board size it would need to also show that out of bounds is invalid. In this case: -1 and 9 either return invalid board signifiers or throw an exception (depending on the practice of the codebase).